core/server/api_container/server/startosis_engine/startosis_interpreter.go

```diff
@@ -386,6 +391,16 @@ func (interpreter *StartosisInterpreter) buildBindings(
 	return &predeclared, nil
 }
 
+const (
+	moduleIdSeperator     = "/"
```

Comment:
Thought: wouldn't this constant already be declared somewhere in the Startosis engine? Should we use that instead, so we don't have two constants doing the same thing?

Reply:
ah good point, ye its declared in git package content provider